app/models/propertyrepresentation/AppointNewAgentSession.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.propertyrepresentation
+
+import play.api.libs.json.{Json, OFormat}
+
+case class AppointNewAgentSession(agentCode: Long, agentOrganisationId: Long, agentOrganisationName: Option[String] = scala.None, isCorrectAgent: Option[Boolean] = scala.None, managingProperty: Option[String] = scala.None)
```

Comment:
scala.None or just None - did you run 'sbt precommit'?

Reply:
for some reason, the compiler is picking up models.propertyrepresentation.None instead of scala.None ...no i didn't run precommit 🤦‍♂  Im  doing that now